ui-tests/pom.xml

```diff
@@ -33,14 +33,29 @@
         <artifactId>jenkins-core</artifactId>
         <version>${jenkins.version}</version>
       </dependency>
+	<!-- need to match any BOM imports from ATH due to https://issues.apache.org/jira/browse/MNG-5761 -->
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-bom</artifactId>
+        <version>4.1.91.Final</version>
```

Comment:
version was lower than required in #1470  care needs to be taken when updating acceptnace-test-harness to keep these aligned (or higher) due to  https://issues.apache.org/jira/browse/MNG-5761